README.md

```diff
@@ -211,15 +215,19 @@ The relayer is configured via a JSON file, the path to which is passed in via th
 
   `"vm": string`
 
-  - The VM type of the source Subnet.
+  - The VM type of the source blockchain.
 
   `"rpc-endpoint": string`
 
-  - The RPC endpoint of the destination Subnet's API node.
+  - The RPC endpoint of the destination blockchains's API node.
 
   `"account-private-key": string`
 
-  - The hex-encoded private key to use for signing transactions on the destination Subnet. May be provided by the environment variable `ACCOUNT_PRIVATE_KEY`. Each `destination-subnet` may use a separate private key by appending the blockchain ID to the private key environment variable name, for example `ACCOUNT_PRIVATE_KEY_11111111111111111111111111111111LpoYY`
+  - The hex-encoded private key to use for signing transactions on the destination blockchain. May be provided by the environment variable `ACCOUNT_PRIVATE_KEY`. Each `destination-subnet` may use a separate private key by appending the cb58 encoded blockchain ID to the private key environment variable name, for example `ACCOUNT_PRIVATE_KEY_11111111111111111111111111111111LpoYY`
+
+  `"allowed-destination-addresses": []string`
+
+  - List of addresses that the relayer is allowed to deliver Warp messages to on this destination blockchain. The destination address is defined by the message protocol. For example, it could be the address called from the message protocol contract. If empty, then all addresses are allowed.
```

Comment:
```suggestion
  - List of addresses that the relayer supports delivering Warp messages to on this destination blockchain. The destination address is defined by the message protocol. For example, it could be the address called from the message protocol contract. If empty, then all addresses are allowed.
```